component/site/libraries/jeventcal.php

```diff
@@ -379,9 +379,10 @@ function getCategoryImage($multiple=false){
 				$output = "";
 				foreach ($data as $cat){
 					$params = json_decode($cat->params);
-					if (isset($params->image) && $params->image!=""){ 
-						$output .= "<img src = '".JURI::root().$params->image."' class='catimage'  alt='categoryimage' />";
-					}							
+					if (isset($params->image) && $params->image!=""){
+						$alt_text = ($params->image_alt == '') ? JText::_('JEV_CAT_ALT_DEFAULT_TEXT') : $params->image_alt;
```

Comment:
This should be javascript safe to esecape any quotes in the alt attribute use JText::_("TO_TRANSLATE",true)